docs/content/6.resources/1.external-blogs-guides-examples.md

```diff
@@ -0,0 +1,10 @@
+# External Blogs, Guides and Examples
+
+This is a collection of resources that we'll expand over time:
+
+-   [Example of Nuxt 3, Strapi and `nuxt-auth`](https://darweb.nl/foundry/article/nuxt3-sidebase-strapi-user-auth)
+-   [Example NuxtAuthHandler configuration for Directus](https://gist.github.com/madsh93/b573b3d8f070e62eaebc5c53ae34e2cc)
+-   [How to expand the user data object](https://github.com/sidebase/nuxt-auth/issues/61#issuecomment-1330946022)
+-   [Docus NuxtAuthHandler that shows how to manually fetch an external JWT token, how to expand the user data and how to track session expiry manually](https://gist.github.com/madsh93/b573b3d8f070e62eaebc5c53ae34e2cc)
+
+Note: If you're looking to make something work in `nuxt-auth` the nice thing is that you can use most `NextAuth.js` resources to help you along the way, as the `NuxtAuthHandler` behaves identical to the [`NextAuth` handler](https://next-auth.js.org/configuration/initialization).
```

Comment:
this file was also auto-formatted (compare it to the old one below) - please revert this / make sure that you just copy paste it

Reply:
Fixed!